### PR TITLE
Enhancements, updates, and fixes

### PR DESCRIPTION
**Pull request by sayandey18**
Update the website, and do various modifications. (SAML, MDX)

### CODE REVIEW

**Review by sayandey18 — COMMENTED** [review]
👍 Positive Feedback:

Include Facebook Metrics to Dashboard (b838f4a, 9fa733f, 6c7734e, b05db5a):

Nice addition of Facebook metrics to the dashboard. It enhances our monitoring capabilities.
Add Website Legal Policies (1506048):

Legal policies are crucial. Thanks for adding them to the website.
Update GitHub Repo Banner (3e0c850, dde5921, b374dc2):

The updated banners look great, improving the overall aesthetics of the GitHub repository.
Update Outdated Content (202cacd, 83a82b2):

Keeping the content up-to-date is essential. Thanks for addressing outdated information.
Update Gmail SMTP with Brevo (7773f73):

Keeping email configurations up-to-date is crucial for proper functioning. Good job on updating the Gmail SMTP settings.
Update Sitemap.xml (c83c62b):

A well-maintained sitemap is essential for search engine optimization. Thanks for updating it.
Include Multiple SAML Providers (b2cd42d, 221cc13):

Adding support for multiple SAML providers is a valuable enhancement.
Fix the Broken Layout (6bd94e5):

Thank you for addressing and fixing the broken layout. It greatly improves the user experience.
Removed Tweets from Website (ffaa9a4, 920cbad):

If the removal of tweets aligns with the project's goals, this is a reasonable change.
Update Packages (90d76c0):

Keeping dependencies up-to-date is essential for security and performance. Good practice.
Update the Redirects (fa478f0):

Properly handling redirects is crucial for maintaining a good user experience. Thanks for updating them.
Update Schema Structure (c740b24):

Maintaining an up-to-date schema structure is crucial. Good job on the update.
Fix Image.remotePatterns Warning (35d3792):

File: .env.example [hunk 1]
```diff
@@ -9,27 +9,38 @@ GOOGLE_API_KEY=
...
+FACEBOOK_ACCESS_TOKEN=
+
 // For MailerLite newsletters
 MAILERLITE_API_KEY=
 MAILERLITE_GROUP_ID=
 
 // For Gmail SMTP Email
-GMAIL_USERNAME=
-GMAIL_PASSWORD=
+SMTP_HOSTNAME=
```
Comment (sayandey18) [inline .env.example:20]: Replace Gmail SMTP with Brevo

**Review by sayandey18 — COMMENTED** [review]
Replace Gmail SMTP with Brevo

**Review by sayakmondal18 — APPROVED** [review]
🤔 Suggestions/Concerns:

If applicable, consider providing more details or context in the commit messages for better understanding.
Confirm that the removal of tweets aligns with the overall content strategy or project goals.
💡 Additional Comments:

Great overall progress on the project. The updates and additions contribute positively to the codebase.
🚀 Next Steps:

If there are no additional concerns, this looks good to merge once any requested changes are addressed.
Thanks for your hard work! 🚀

**Pull request merged**